test/Microsoft.AspNet.Mvc.Core.Test/ControllerTests.cs

```diff
@@ -1297,6 +1297,220 @@ public void ControllerView_UsesControllerViewEngine()
             Assert.Same(viewEngine, result.ViewEngine);
         }
 
+        [Theory]
+        [MemberData(nameof(TestabilityViewTestData))]
+        public void ControllerView_InvokedInUnitTests(object modelObject, string viewName)
+        {
+            // Arrange
+            var controller = new TestabilityController();
+
+            // Act
+            var result = (ViewResult)controller.View_Action(viewName, modelObject);
+
+            // Assert
+            Assert.NotNull(result);
+            Assert.NotNull(result.ViewData);
+            Assert.Equal(viewName, result.ViewName);
+
+            if (modelObject != null)
+            {
+                var expectedObject = (MyModel)modelObject;
+                Assert.Equal(expectedObject.Property1, ((MyModel)result.ViewData.Model).Property1);
+                Assert.Equal(expectedObject.Property2, ((MyModel)result.ViewData.Model).Property2);
```

Comment:
There should probably also be an `Assert.Same(controller.ViewData, result.ViewData)`